plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/bedrock-runtime.ts

```diff
@@ -94,6 +96,144 @@ export class BedrockRuntimeServiceExtension implements ServiceExtension {
     };
   }
 
+  private requestPreSpanHookInvokeModel(
+    request: NormalizedRequest,
+    config: AwsSdkInstrumentationConfig,
+    diag: DiagLogger
+  ): RequestMetadata {
+    let spanName: string | undefined;
+    const spanAttributes: Attributes = {
+      [ATTR_GEN_AI_SYSTEM]: GEN_AI_SYSTEM_VALUE_AWS_BEDROCK,
+      // add operation name for InvokeModel API
+    };
+
+    const modelId = request.commandInput.modelId;
```

Comment:
Let's be consistent with other usages of `request.commandInput`, even if the TypeScript type says `commandInput` is defined.
```suggestion
    const modelId = request.commandInput?.modelId;
```